client/finality-grandpa/src/communication/gossip.rs

```diff
@@ -110,7 +110,7 @@ const CATCH_UP_THRESHOLD: u64 = 2;
 const PROPAGATION_ALL: u32 = 4; //in rounds;
 const PROPAGATION_ALL_AUTHORITIES: u32 = 2; //in rounds;
 const PROPAGATION_SOME_NON_AUTHORITIES: u32 = 3; //in rounds;
-const ROUND_DURATION: u32 = 4; // measured in gossip durations
+const ROUND_DURATION: u32 = 2; // measured in gossip durations
```

Comment:
Did the previous value trigger incorrect behaviour? I see it's used in two places below for checking if messages are allowed?

Reply:
No, it meant that we were filtering gossip messages for longer than we expected (e.g. we propagate to all peers after `ROUND_DURATION * PROPAGATION_ALL`), the actual behavior shouldn't change though. A GRANDPA round takes two timeout turns, i.e. prevote and precommit, and not 4.